x-pack/solutions/security/plugins/security_solution/public/detections/components/alert_summary/wrapper.tsx

```diff
@@ -61,30 +63,23 @@ export interface WrapperProps {
  * If the creation fails, we show an error message.
  */
 export const Wrapper = memo(({ packages, ruleResponse }: WrapperProps) => {
-  const { data } = useKibana().services;
-  const [dataView, setDataView] = useState<DataView | undefined>(undefined);
-  const [loading, setLoading] = useState<boolean>(true);
+  const spaceId = useSpaceId();
+  const dataViewSpec = useMemo(() => ({ title: `${DEFAULT_ALERTS_INDEX}-${spaceId}` }), [spaceId]);
 
-  useEffect(() => {
-    let dv: DataView;
-    const createDataView = async () => {
-      try {
-        dv = await data.dataViews.create(dataViewSpec);
-        setDataView(dv);
-        setLoading(false);
-      } catch (err) {
-        setLoading(false);
-      }
-    };
-    createDataView();
+  const signalIndexName = `${DEFAULT_ALERTS_INDEX}-${spaceId}`;
```

Comment:
very minor nit (feel free to ignore it) but you're using `${DEFAULT_ALERTS_INDEX}-${spaceId}` twice here, just switch the 2 lines around and use `signalIndexName` when you create the `dataViewSpec` constant

Reply:
No idea why I missed that! thank you! updated